### PR TITLE
B/unnest gravatar

### PR DESCRIPTION
**Pull request by spasiu**

### CODE REVIEW

File: source/includes/_configuring_agent_images.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Configuring agent images
+You can set an agent's image in your app (as pictured below) by creating a <a href="http://www.gravatar.com/">Gravatar</a> image associated with the agent's email address.
```
Comment (mspensieri) [inline source/includes/_configuring_agent_images.md]: The word "agent" doesn't really make sense anymore if we're not in the context of Zendesk

File: source/includes/_configuring_agent_images.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Configuring agent images
```
Comment (mspensieri) [inline source/includes/_configuring_agent_images.md]: This should probably be h1 or h2 `#` `##`

File: source/includes/_configuring_agent_images.md (hunk 1 above)
Comment (spasiu) [inline source/includes/_configuring_agent_images.md]: what do we call them? App maker? Collaborator?

Comment (mspensieri) [inline source/includes/_configuring_agent_images.md]: app maker is the best we have atm I think

**Comment by mspensieri** [comment]
:+1:

**Pull request merged**